main/acle.rst

```diff
@@ -5822,3 +5835,183 @@ Instructions
+| .. code:: c                                                                    |
+|                                                                                |
+|      svint8_t svset_neonq[_s8](svint8_t vec, int8x16_t subvec)                 |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svint16_t svset_neonq[_s16](svint16_t vec, int16x8_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svint32_t svset_neonq[_s32](svint32_t vec, int32x4_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svint64_t svset_neonq[_s64](svint64_t vec, int64x2_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint8_t svset_neonq[_u8](svuint8_t vec, uint8x16_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint16_t svset_neonq[_u16](svuint16_t vec, uint16x8_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint32_t svset_neonq[_u32](svuint32_t vec, uint32x4_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint64_t svset_neonq[_u64](svuint64_t vec, uint64x2_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat16_t svset_neonq[_f16](svfloat16_t vec, float16x8_t subvec)        |
```

Comment:
I think there should be a bfloat16 version too.  For SVE, this kind of _bf16 function is available unconditionally.

Reply:
done.